css/components/card.css

```diff
@@ -38,6 +38,27 @@ Actions contains:-Button and Icons*/
 }
 
 
+/* For Only Text cards*/
+.card_only_text{
+    height: fit-content;
+}
+.card_with_dismiss{
+    position: relative;
+}
+
+.card_dismiss{
+    padding: 0.4rem 0.2rem;
```

Comment:
you can use px for less than 1rem

Reply:
Thanks, good suggestion. I have used this at many places. Will do these changes in refactoring pr